spec/models/manageiq/providers/amazon/storage_manager/ebs/cloud_volume_spec.rb

```diff
@@ -35,6 +35,139 @@
       end
     end
 
+    context "#update_volume" do
+      let(:modify_volume_stub) { { :modify_volume => {} } }
+
+      context "updating volume name" do
+        it "is allowed when :name is provided" do
+          # Name is set via AWS tags so stub the create_tags.
+          expect_any_instance_of(Aws::EC2::Volume).to receive(:create_tags).with(
+            :tags => [{:key => "Name", :value => "new_name"}]
```

Comment:
Possible, but only to some degree. I wanted to ensure that specific arguments are used as well as check whether the call is made or not (if name is given, it must be called, if it's not given, it should not be called).

What I could do is ignore input parameters to `create_tags` and then use an exception in the stub to ensure it doesn't get called. I'll have a look.